cmd/zqd/listen/command.go

```diff
@@ -136,20 +138,28 @@ func pprofHandlers(h http.Handler) http.Handler {
 //     name: "http.access"
 //     level: info
 //     mode: truncate
+// sortMemMaxBytes: 268432640
 
 func (c *Command) loadConfigFile() error {
 	if c.configfile == "" {
 		return nil
 	}
 	conf := &struct {
-		Logger logger.Config `yaml:"logger"`
+		Logger          logger.Config `yaml:"logger"`
+		SortMemMaxBytes *int          `yaml:"sortMemMaxBytes,omitempty"`
 	}{}
 	b, err := ioutil.ReadFile(c.configfile)
 	if err != nil {
 		return err
 	}
 	err = yaml.Unmarshal(b, conf)
 	c.loggerConf = &conf.Logger
+	if v := conf.SortMemMaxBytes; v != nil {
+		if *v <= 0 {
+			return fmt.Errorf("%s: sortMemMaxBytes value must be greater than zero", c.configfile)
+		}
+		proc.SortMemMaxBytes = *v
```

Comment:
I suppose this is ok for now, but ideally this would be passed into the zqd's core.Config. I always worry about these global w.r.t running unit tests in parallel.

Reply:
Agreed. There has to be a better way. I'm just not yet sure what that is yet.